My-first-widget tutorial: split materials (#395)
ewoks-kit/ewoksorange: doc/tutorials/my_first_widget/display_output.rst

From orange main area to control area
"""""""""""""""""""""""""""""""""""""

.. comment::

>     for the following code-block this is not the full diff. But only a subset. This is why we are force to have it in "raw".

Reviewer: ```suggestion
    The following code-block is only a subset of the full diff. This is why it is displayed in raw format.
```
